File: Claims-management-System-Frontend/src/app/claim.ts

export class Claim {
    claimId! : number;
	policyId! : number;
	memberId! : number;
	providerId! : number;
	benefitId! : number;
	totalBilledAmount! : DoubleRange;
	totalClaimedAmount! : DoubleRange;
	claimStatus! : string;
	claimDescription! : string;
}

File: Claims-Management-System-Backend/ClaimsMicroservice/src/main/java/com/cognizant/claimsmicroservice/model/Claim.java
package com.ayushi.claimsmicroservice.model;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import org.springframework.stereotype.Component;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "claim")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Component
public class Claim {

	@Id
	private int claimId;
	private String claimStatus;
	private String description;
	private int policyId;
	private int memberId;
	private int benefitId;
	private int providerId;
	private double amountClaimed;
	private double amountSettled;
	
	
	
}
